### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by rondlite** [reactioncommerce/api-migrations#30]
The following dependencies are missing when I run migrate:

-  path-parser
-  lodash._baseiteratee
-  lodash._baseuniq

**Comment by MohanNarayana** [comment]
@rondlite I will work on the PR once the DCO is signed off. May I know how you came across this issue? I was recently working on creating a migration for a api-plugin-tags and did not see the same. Just curious and also helpful for reproducing the issue for testing.

**Comment by rondlite** [comment]
@MohanNarayana I am trying to figure out how to change the submitter, since the commit is signed of by me, but I committed it from a dev machine which I did not realize was linked to another account, so the DCO does not match the account email.
Wondering if I should just remove this PR and make a new one from the right account, since I can amend a signature but changing committer is not possible afaik.

I came across this issue when I was moving dev server to staging and this triggered an error about my version not right, so I ran migrate and got the error. Weird since they are not packages one would have installed globally, since can't think of other reason you would not get the missing dependencies.

**Comment by MohanNarayana** [comment]
@rondlite if there is no issue signing off with work account you should do that. If you have any restriction with that then I guess a new PR is the easiest way.

**Comment by DemandDevOps** [comment]
I guess I make new PR since running into new issue when trying to commit from work account (i am cto so yes, it was an option).
Was following DCO bot path
```
git push --force-with-lease origin missing-dependencies
Username for 'https://github.com': DemandDevOps
Password for 'https://DemandDevOps@github.com': 
remote: Permission to reactioncommerce/api-migrations.git denied to DemandDevOps.
fatal: unable to access 'https://github.com/reactioncommerce/api-migrations.git/': The requested URL returned error: 403
```

**Comment by rondlite** [comment]
didn't have restrictions but ran into permissions :wink: 
redone the PR

**Comment by MohanNarayana** [comment]
Fixed in #33
